app/components/verkiezingen/prepare-legislatuur-section.hbs

```diff
@@ -0,0 +1,54 @@
+{{#if (await this.isBurgemeester)}}
+  <Mandaat::BurgemeesterSelector @bestuursorgaanInTijd={{@bestuursorgaan}} />
+{{else}}
+  <AuToolbar
+    @size="large"
+    @nowrap="true"
+    class="au-u-padding-bottom-none au-u-margin-bottom-none"
+    as |Group|
+  >
+    <Group class="au-u-margin-bottom-none">
+      <div class="au-o-flow">
+        <AuHeading @skin="3">Samenstelling
+          {{@bestuursorgaan.isTijdsspecialisatieVan.naam}}</AuHeading>
+      </div>
+    </Group>
+  </AuToolbar>
+  <Verkiezingen::DraftMandatarisList
+    @bestuursorgaan={{@bestuursorgaan}}
+    @sort={{@sort}}
+    @page={{0}}
+    @size={{9999}}
+    @title=""
+    @showFractie={{true}}
+    @showFunctie={{true}}
+    @showRangorde={{true}}
+    @showBevoegdheid={{true}}
+    @showStartEndDate={{true}}
+  />
+
+  <AuContent @skin="small">
+    <AuButton
+      @skin="naked"
+      @icon="add"
+      {{on "click" this.createMandataris}}
+    >Toevoegen</AuButton>
```

Comment:
This is not for this PR but should we find a way that we can add the column in the component itself and only refresh the component?

Reply:
baby steps...